safe/gis/vector/summary_5_multi_exposure.py

```diff
@@ -0,0 +1,185 @@
+# coding=utf-8
+
+"""Multi-exposure summary calculation."""
+
+from qgis.core import QgsFeatureRequest
+
+from safe.definitions.fields import (
+    hazard_count_field,
+    exposure_hazard_count_field,
+    total_affected_field,
+    exposure_total_affected_field,
+    total_not_affected_field,
+    exposure_total_not_affected_field,
+    total_exposed_field,
+    exposure_total_exposed_field,
+    total_not_exposed_field,
+    exposure_total_not_exposed_field,
+    total_field,
+    exposure_total_field,
+)
+from safe.gis.vector.tools import (
+    create_field_from_definition,
+    read_dynamic_inasafe_field,
+)
+
+__copyright__ = "Copyright 2017, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+def multi_exposure_analysis_summary(analysis, intermediate_analysis):
```

Comment:
I saw in this function you merge the analysis layers into one. What about the metadata? Especially the provenance

Reply:
I'm just merging QGIS layers. The same as summary_X.py in the gis/vector package.
We still need to know/do about provenance.